### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by ArmorDarks** [ddluc/grunt-backstop#5]
BacktstopJS from now can be used as Node module, thus there is no need to have configure path to bower's components, force configuration and so on.

Are there any chances for an update?

**Comment by ddluc** [comment]
Yeah, it's possible. I'm pretty swamped right now with life/work so not sure when I will have time make any fixes. Did the update to backstop cause my grunt task to break, or does it just need updated for constancy?
